lib/filter-org-members-action.js

```diff
@@ -0,0 +1,64 @@
+const core = require('@actions/core')
+const github = require('@actions/github')
+
+run()
+
+async function run() {
+
+    try {
+
+        const api = github.getOctokit(core.getInput("token", { required: true }), {})
+
+        const organization = core.getInput("organization")
+        const usernames = core.getInput("usernames").split(",").map(s => s.trim())
+
+        console.log(`Will check if input users belong to org ${organization}`)
+
+        const query = `query($cursor: String, $org: String!, $userLogins: [String!], $username: String!)  {
+            user(login: $username) {
+                id
+            }
+            organization(login: $org) {
+              teams (first:1, userLogins: $userLogins, after: $cursor) {
+                  nodes {
```

Comment:
What does `nodes` mean?

Reply:
`nodes` is a collection of resources. 

For here, you get a collection of `teams` the user belongs to.